### DISCUSSION TITLE
建议添加123盘直链功能，长期白嫖不稳定！

### DISCUSSION THREAD

**Discussion by jaman21** [AlistGo/alist#5140] [General]
自定义302重定向到 https://vip.123pan.cn/用户ID/

**Comment by Troray** [comment]
alist的`地址树(URLTREE)`可添加直链。

**Reply by jaman21** [reply]
直连地址是没有文件列表的，只能指定文件才能获取。Urltree不知道怎么合理设置？感觉需要与123盘驱动集成。

**Comment by pongfcnkl** [comment]
去让123官方开放api

**Reply by 520yxl** [reply]
问过了,官方不考虑开放api
